File: Praktikum1/PRAK105-2010817210023-MuhammadAuliaAkbar.c
#include <stdio.h>

int main()
{
    int a, b, x, y;
    printf("Masukan Nilai A: ");
    scanf("%d", &a);
    printf("Masukan Nilai B: ");
    scanf("%d", &b);
    printf("Masukan Nilai X: ");
    scanf("%d", &x);
    printf("Masukan Nilai Y: ");
    scanf("%d", &y);
    int hasilbagiab = a % b;
    int hasilbagixy = x % y;
    int jumlah = hasilbagiab + hasilbagixy;
    printf("Total sisa hasil bagi dari %d dibagi %d dan %d dibagi %d adalah %d", a, b, x, y, jumlah);
    return 0;
}

File: Praktikum1/PRAK105-2010817210023-MuhammadAuliaAkbar.py
a = int(input("Masukan Nilai A: "))
b = int(input("Masukan Nilai B: "))
x = int(input("Masukan Nilai X: "))
y = int(input("Masukan Nilai Y: "))

hasilbagiab = a % b
hasilbagixy = x % y
jumlah = hasilbagiab + hasilbagixy

print("Total sisa hasil bagi dari {} dibagi {} dan {} dibagi {} adalah {}".format(a, b, x, y, jumlah))
